server.js

```diff
@@ -1,4 +1,4 @@
-import express from 'express';
+var express = require('express');
```

Comment:
This shouldn't be necessary since the top-level `start.js` file registers the Babel hook before `require`ing server.